__tests__/integration/mirador/basic.test.js

```diff
@@ -7,15 +7,15 @@ describe('Basic end to end Mirador', () => {
     expect(title).toBe('Mirador');
   });
   it('loads a manifest and displays it', async () => {
-    await expect(page).toFill('#manifestURL', 'https://purl.stanford.edu/sn904cj3429/iiif/manifest');
+    await expect(page).toFill('#manifestURL', 'http://localhost:5000/sn904cj3439.json');
     await expect(page).toClick('#fetchBtn');
     // TODO: Refactor the app so we get rid of the wait
     await page.waitFor(1000);
-    await expect(page).toMatchElement('li', { text: 'https://purl.stanford.edu/sn904cj3429/iiif/manifest' });
+    await expect(page).toMatchElement('li', { text: 'http://localhost:5000/sn904cj3439.json' });
     await expect(page).toMatchElement(
       'h3',
       "Peter's San Francisco Locator. The Birds-Eye-View Map of the Exposition City. Published by Locator Publishing Co",
     );
     await expect(page).toMatchElement('div', /Color/);
-  });
+  }, 30000);
```

Comment:
One quick question about this, I see this added in this test, but not in the others. Is this needed here?